proto/src/determined/master/v1/master.proto

```diff
@@ -87,7 +87,4 @@ message ResourceAllocationAggregatedEntry {
   // The seconds in the cluster used by experiments assigned to each resource
   // pool.
   map<string, float> by_resource_pool = 6;
-  // The seconds in the cluster used by experiments assigned to each agent
-  // label.
-  map<string, float> by_agent_label = 7;
```

Comment:
ditto on leaving the protos alone (I think... I'm less sure about the implications of this instance.  If it only affects the webui, which ships with the master, then it's fine).